tests/plugins/tabletools/cellproperties.js

```diff
@@ -1,5 +1,5 @@
 /* bender-tags: editor */
-/* bender-ckeditor-plugins: entities,dialog,tabletools,toolbar */
+/* bender-ckeditor-plugins: entities,dialog,tabletools,toolbar,tableselection */
```

Comment:
You are only using existing testing function, any reason to add new plugin?

Reply:
There was - tests were failing in built full-all version and the reason was `tableselection` plugin. Therefore I added it to tests to be sure if it's working correctly. I fixed the issue in the next commit, so right now plugin is not necessary, it can be removed.